cabal-install/Distribution/Client/ProjectConfig/Legacy.hs

```diff
@@ -1402,6 +1402,16 @@ newLineListField :: String -> (a -> Doc) -> ReadP [a] a
                  -> (b -> [a]) -> ([a] -> b -> b) -> FieldDescr b
 newLineListField = listFieldWithSep Disp.sep
 
+multiListField :: String -> (a -> Doc) -> ReadP [a] a
```

Comment:
newLineListField has a comment, `

> like
-- commaNewLineListField it pretty prints on multiple lines

ins't it working? What does this PR changes? Examples, tests missing.